src/modules/land_detector/LandDetector.cpp

```diff
@@ -58,11 +58,15 @@ LandDetector::~LandDetector()
 void LandDetector::start()
 {
 	_update_params();
-	ScheduleOnInterval(LAND_DETECTOR_UPDATE_INTERVAL);
+	ScheduleDelayed(20_ms);
+	_vehicle_local_position_sub.registerCallback();
 }
 
 void LandDetector::Run()
 {
+	// push backup schedule
+	ScheduleDelayed(20_ms);
```

Comment:
Yes it's pushing the schedule forward (hrt call with a deadline). It would only ever actually result in scheduling the WorkItem if `vehicle_local_position` stopped publishing.